src/plays/cricket-game/styles.css

```diff
@@ -0,0 +1,289 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600;700;800;900&family=Signika+Negative:wght@300;400;500;600;700&display=swap');
+
+/* enter styles here */
+
```

Comment:
You gave a class name called `cricket-home-body` already. Please use it to prefix all the class names below.. like `.cricket-home-body img`, `.cricket-home-body .modal`, etc. it is applicable to all the class names in this file.

Reply:
Fixed!
https://github.com/reactplay/react-play/pull/972/commits/c7dba9eedf67f7176c2f523e63b23b6d6720673e